pvlib/pvsystem.py

```diff
@@ -3031,3 +3031,144 @@ def combine_loss_factors(index, *losses, fill_method='ffill'):
         combined_factor *= (1 - loss)
 
     return 1 - combined_factor
+
+
+def nonuniform_irradiance_deline_power_loss(
+    rmad,
+    model: Literal[
+        "fixed-tilt", "single-axis-tracking"
+    ] = "single-axis-tracking",
+    fillfactor_ratio: float = None,
+):
+    r"""
+    Estimate the power loss due to irradiance non-uniformity.
+
+    This model is described for bifacial modules in [1]_, where the backside
+    irradiance is less uniform due to mounting and site conditions.
+
+    Depending on the mounting type, the power loss is estimated with either
+    equation (11) or (12) of [1]_. Passing a custom polynomial is also valid.
+
+    Use ``fillfactor_ratio`` to account for different fill factors between the
+    trained model and the module of interest. The fill factor used for
+    developing the models ``"single-axis-tracking" and "fixed-tilt"`` is
+    :math:`0.79`. For example, if the fill factor of the module of interest is
+    :math:`0.65`, then set ``fillfactor_ratio = 0.65 / 0.79``.
+
+    .. versionadded:: 0.11.0
+
+    Parameters
+    ----------
+    rmad : numeric
+        The Relative Mean Absolute Difference of the irradiance.
```

Comment:
II think we need to be more specific here about `irradiance`. Here, I believe it should be the RMAD of cell-by-cell total (front + rear) irradiance (Eq. 2 of [1]). I presume that `rmad` is a `Series` with a different value each timestamp. But I can't tell from the reference what time resolution is to be applied.  From the paper, I don't know if the model applies for hourly averages (as in their simulations) or once per hour with instantaneous data (as in their laboratory measurements). We could reach out and clarify with the paper authors; they are on github @cdeline @shirubana